pkg/loop/median_service.go

```diff
@@ -0,0 +1,42 @@
+package loop
```

Comment:
is this being moved out the the `chainlink` repo? if so, can we do that in separate change?

Reply:
What do you mean? Separate how?
Actually using this in core is already separate, what difference does it make if this type is included here now?

Reply:
i would like to see the atomicity and generic pluginService in one change and then a second change to refactor the existing Median/Relayer code to use them.

is that possible? i get too lost reading the code to disentangle the moving pieces.

Reply:
I don't think that is possible, not without duplicating most things in the mean time.